wizard/WizardDaemonSettings.qml

```diff
@@ -0,0 +1,198 @@
+// STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
+// THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import moneroComponents.WalletManager 1.0
+import QtQuick 2.2
+import QtQuick.Layouts 1.1
+import "../components"
+import "utils.js" as Utils
+
+ColumnLayout {
+    Layout.leftMargin: wizardLeftMargin
+    Layout.rightMargin: wizardRightMargin
+
+    id: passwordPage
+    opacity: 0
+    visible: false
+    property alias titleText: titleText.text
+    Behavior on opacity {
+        NumberAnimation { duration: 100; easing.type: Easing.InQuad }
+    }
+
+    onOpacityChanged: visible = opacity !== 0
+
+
+    function onPageOpened(settingsObject) {
+    }
+    function onWizardRestarted(){
+    }
+
+    function onPageClosed(settingsObject) {
+        appWindow.persistentSettings.useRemoteNode = remoteNode.checked
+        appWindow.persistentSettings.startLocalNode = localNode.checked
+        appWindow.persistentSettings.remoteNodeAddress = remoteNodeEdit.getAddress();
+        return true
+    }
+
+    RowLayout {
+        id: dotsRow
+        Layout.alignment: Qt.AlignRight
+
+        ListModel {
+            id: dotsModel
+            ListElement { dotColor: "#36B05B" }
+            ListElement { dotColor: "#36B05B" }
+            ListElement { dotColor: "#FFE00A" }
+            ListElement { dotColor: "#DBDBDB" }
+        }
+
+        Repeater {
+            model: dotsModel
+            delegate: Rectangle {
+                // Password page is last page when creating view only wallet
+                // TODO: make this dynamic for all pages in wizard
+                visible: (wizard.currentPath != "create_view_only_wallet" || index < 2)
+                width: 12; height: 12
+                radius: 6
+                color: dotColor
+            }
+        }
+    }
+
+    ColumnLayout {
+        id: headerColumn
+
+        Text {
+            Layout.fillWidth: true
+            id: titleText
+            font.family: "Arial"
+            font.pixelSize: 28 * scaleRatio
+            wrapMode: Text.Wrap
+            horizontalAlignment: Text.AlignHCenter
+            //renderType: Text.NativeRendering
+            color: "#3F3F3F"
+            text: "Daemon settings"
+
+        }
+
+        Text {
+            Layout.fillWidth: true
+            Layout.topMargin: 30 * scaleRatio
+            Layout.bottomMargin: 30 * scaleRatio
+            font.family: "Arial"
+            font.pixelSize: 18 * scaleRatio
+            wrapMode: Text.Wrap
+            //renderType: Text.NativeRendering
+            color: "#4A4646"
+            textFormat: Text.RichText
+//            horizontalAlignment: Text.AlignHCenter
+            text: qsTr("To be able to communicate with the Monero network your wallet needs to be connected to a Monero node. For best privacy it's recommended to run your own node.
+                        <br><br>
```

Comment:
Github renders this in red, leading me to believe it was interpreted (and hidden) some data, so what I'm reviewing may not be what is really in the patch. Any weird thing on this line ?

Reply:
removed 2 unescaped linebreaks that caused the red marking.